x/group/go.mod

```diff
@@ -0,0 +1,140 @@
+module cosmossdk.io/x/group
+
+go 1.21.1
+
+require (
+	cosmossdk.io/api v0.7.1
+	cosmossdk.io/core v0.12.0
+	cosmossdk.io/depinject v1.0.0-alpha.4
+	cosmossdk.io/errors v1.0.0
+	cosmossdk.io/log v1.2.1
+	cosmossdk.io/store v1.0.0-rc.0
+	github.com/cockroachdb/apd/v2 v2.0.2
+	github.com/cometbft/cometbft v0.38.0
+	github.com/cosmos/cosmos-db v1.0.0
+	github.com/cosmos/cosmos-proto v1.0.0-beta.3
+	github.com/cosmos/cosmos-sdk v0.51.0
```

Comment:
Great! You need to add a replace of the SDK here.

Reply:
Done, thanks!